geoportal/c2cgeoportal_geoportal/views/theme.py

```diff
@@ -221,6 +218,9 @@ def build_web_map_service(ogc_server_id):
                     "children": [l.name for l in wms_layer.layers],
                 }
 
+            del wms
+            gc.collect(2)
```

Comment:
Do you really need `gc.collect`? And why only the second pool?

Reply:
Yes to try to free memory behind capabilities parsing... => runs all generations